example/src/main/java/org/apache/rocketmq/example/benchmark/timer/TimerConsumer.java

```diff
@@ -151,6 +166,30 @@ private Options buildCommandlineOptions(Options options) {
         opt.setRequired(false);
         options.addOption(opt);
 
+        opt = new Option("w", "threadCount", true, "Thread count, Default: 20");
+        opt.setRequired(false);
+        options.addOption(opt);
+
+        opt = new Option("m", "msgTraceEnable", true, "Message Trace Enable, Default: false");
+        opt.setRequired(false);
+        options.addOption(opt);
+
+        opt = new Option("g", "group", true, "Consumer group name, Default: benchmark_consumer");
+        opt.setRequired(false);
+        options.addOption(opt);
+
+        opt = new Option("a", "aclEnable", true, "Acl Enable, Default: false");
```

Comment:
how about use a no argument option ?

Reply:
Good suggestion, but to keep consistent wist other code, I don't do any change here.
 If It really need to change, I'd like to make a new patch to change all .sh .